### PR TITLE
`2128` ensure Role Info form validates correctly

### LINKED ISSUE #2128: Role form validation not working

### PR DESCRIPTION
**Pull request by Da-Colon**
Fixes #2128

### CODE REVIEW

**Review by DarksightKellar — APPROVED** [review]
Nice

File: src/components/pages/Roles/forms/RoleFormInfo.tsx [hunk 1]
```diff
@@ -20,19 +20,14 @@ export default function RoleFormInfo() {
...
-          {({
-            field,
-            form: { setFieldValue, setFieldTouched },
-            meta,
-          }: FieldProps<string, RoleFormValues>) => (
+          {({ field, form: { setFieldValue }, meta }: FieldProps<string, RoleFormValues>) => (
             <LabelWrapper
               label="Role Name"
-              errorMessage={meta.touched && meta.error ? meta.error : undefined}
+              errorMessage={meta.error ? meta.error : undefined}
```
Comment (adamgall) [inline src/components/pages/Roles/forms/RoleFormInfo.tsx:26]: what type is `meta.error`?

**Review by adamgall — APPROVED** [review]
Giving an approval, but curious about what type `meta.error` is because why not just do `errorMessage={meta.error}`?

File: src/components/pages/Roles/forms/RoleFormInfo.tsx (hunk 1 above)
Comment (Da-Colon) [reply src/components/pages/Roles/forms/RoleFormInfo.tsx]: Good call, its `string | undefined`

**Pull request merged**